packages/kyt-core/package.json

```diff
@@ -72,5 +74,10 @@
     "kyt",
     "core",
     "kyt-core"
-  ]
+  ],
+  "devDependencies": {
+    "react": "^15.4.2",
+    "react-addons-test-utils": "^15.4.2",
+    "react-dom": "^15.4.2"
+  }
```

Comment:
Had to add these because of enzyme-to-json serializer. It's a devDependency so it shouldn't affect anything in the published packages.